ctwrap/simulation.py

```diff
@@ -87,17 +90,41 @@ def defaults(self):
         """Pass-through returning module defaults as a dictionary"""
         return self._module.defaults()
 
-    def _save(self):
-        """Save simulation data (hidden)"""
+    def _save(self, mode='a'):
```

Comment:
The `_save` method should be removed - now that `write_hdf` is part of Cantera, saving everything in the default HDF format is much preferable: per previous discussion, this makes things really simple recreating objects.

I.e. `write_hdf` should be used within the module, giving a user full freedom on how to save.